02-Python-1.ipynb

```diff
@@ -0,0 +1,844 @@
+    "fig = matplotlib.pyplot.figure(figsize=(10.0, 3.0))\n",
+    "\n",
+    "axes1 = fig.add_subplot(1, 3, 1)\n",
+    "axes2 = fig.add_subplot(1, 3, 2)\n",
+    "axes3 = fig.add_subplot(1, 3, 3)\n",
+    "\n",
+    "axes1.set_ylabel('average')\n",
+    "axes1.plot(data.mean(axis=0))\n",
+    "\n",
+    "axes2.set_ylabel('max')\n",
+    "axes2.plot(data.max(axis=0))\n",
+    "\n",
+    "axes3.set_ylabel('min')\n",
+    "axes3.plot(data.min(axis=0))\n",
+    "\n",
+    "fig.tight_layout()\n",
+    "\n",
+    "matplotlib.pyplot.show(fig)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The call to `loadtxt` reads our data, and the rest of the program tells the plotting library how large we want the figure to be, that we’re creating three sub-plots, what to draw for each one, and that we want a tight layout. (Perversely, if we leave out that call to `fig.tight_layout()`, the graphs will actually be squeezed together more closely.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<section class=\"objectives panel panel-warning\">\n",
+    "<div class=\"panel-heading\">\n",
+    "<h2><span class=\"fa fa-certificate\"></span> Scientists dislike typing </h2>\n",
+    "</div>\n",
+    "<br/>\n",
+    "We will always use the syntax `import numpy` to import NumPy. However, in order to save typing, it is often suggested to make a shortcut like so: `import numpy as np`. If you ever see Python code online using a NumPy function with `np` (for example, `np.loadtxt(...)`), it’s because they’ve used this shortcut.\n",
+    "\n",
+    "</section>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<section class=\"objectives panel panel-success\">\n",
+    "<div class=\"panel-heading\">\n",
+    "<h2><span class=\"fa fa-pencil\"></span> Make your own plot </h2>\n",
+    "</div>\n",
+    "<br/>\n",
+    "What does the following program print out?\n",
+    "\n",
+    "```python\n",
+    "first, second = 'Grace', 'Hopper'\n",
+    "third, fourth = second, first\n",
+    "print third, fourth\n",
+    "```\n",
+    "\n",
+    "\n",
+    "</section>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<section class=\"objectives panel panel-success\">\n",
+    "<div class=\"panel-heading\">\n",
+    "<h2><span class=\"fa fa-pencil\"></span> Slicing strings </h2>\n",
+    "</div>\n",
+    "\n",
+    "A section of an array is called a slice. We can take slices of character strings as well:\n",
+    "\n",
+    "```python\n",
+    "element = 'oxygen'\n",
+    "print('first three characters:', element[0:3])\n",
+    "print('last three characters:', element[3:6])\n",
+    "```\n",
+    "What is the value of element[:4]? What about element[4:]? Or element[:]?\n",
+    "\n",
+    "What is element[-1]? What is element[-2]? Given those answers, explain what element[1:-1] does.\n",
+    "\n",
+    "\n",
+    "</section>"
```

Comment:
If you move `</section> up to line 733, then the sintax highlighting will work... otherwise the python code sample will get wrapped around with the other text
Same with the previous exercise